### PR TITLE
Don't include BreadcrumbsOnRails into the ActionController::API

### PR DESCRIPTION
**Pull request by jcoyne**
Alternative proposal combining a test with the fix from #77

### CODE REVIEW

**Comment by jcoyne** [comment]
It looks like this fix isn't going to work on ruby 2.0.x. because include is private there. Can you merge #95 first and then I can rebase this?

**Comment by weppos** [comment]
Removing support for Ruby and/or Rails versions requires a major bump, as I want to be consistent with SemVer.

I propose to use send(:include) that will bypass the limit for this version, so that I can package a 3.0.1.

After that, I'm fine to merge #95 and drop support for previous versions. Please note that #95 also requires an update to the README and `.gemspec`

**Comment by jcoyne** [comment]
I've added `send :include, ...`

**Comment by jcoyne** [comment]
I wonder if this will hit the `Cannot define multiple 'included' blocks for a Concern` in production mode?

**Comment by jcoyne** [comment]
It doesn't seem to be a problem. I tested this change in production mode.

File: lib/breadcrumbs_on_rails/railtie.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ module BreadcrumbsOnRails
 
   class Railtie < Rails::Railtie
     ActiveSupport.on_load(:action_controller) do
-      include BreadcrumbsOnRails::ActionController
+      ::ActionController::Base.send :include, BreadcrumbsOnRails::ActionController
```
Comment (jonathansimmons) [inline lib/breadcrumbs_on_rails/railtie.rb]: This is nit picky but I think most would agree we should use parenthesis here: https://github.com/bbatsov/ruby-style-guide#method-invocation-parens

**Comment by jcoyne** [comment]
Added parens

**Pull request merged**